concepts/packages/.meta/config.json

```
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb",
-  "authors": ["ErikSchierboom"],
+  "blurb": "Packages are the unit of code reuse in Go",
+  "authors": ["ErikSchierboom", "bobtfish"],
```

I think I should probably be removed as an author:

```suggestion
  "authors": ["bobtfish"],
```